app/controllers/v0/sessions_controller.rb

```diff
@@ -88,5 +85,17 @@ def delete_session(token)
       end
     end
     # :nocov:
+
+    def log_errors
+      message = <<-MESSAGE.strip_heredoc
```

Comment:
@saneshark i followed your example for preserving tabs, whitespace & newlines [from here](https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/common/exceptions/external/backend_service_exception.rb#L75), did I get it right?  heredoc is a new concept for me.